Fix CI

```sh
Cloning into 'zombodb'...
Error: 
   0: `pgrx-0.9.3` shouldn't be used with `cargo-pgrx-0.8.3`, please use `pgrx = "~0.8.3"` in your `Cargo.toml`.

Location:
   /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-pgrx-0.8.3/src/metadata.rs:42

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ SPANTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

   0: cargo_pgrx::metadata::validate
      at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-pgrx-0.8.3/src/metadata.rs:27
   1: cargo_pgrx::command::install::execute
      at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-pgrx-0.8.3/src/command/install.rs:51

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
The command '/bin/sh -c apk add --no-cache --virtual .zombodb-build-deps     git 	curl 	bash 	ruby-dev 	ruby-etc 	musl-dev 	make 	gcc 	coreutils 	util-linux-dev 	musl-dev 	openssl-dev     clang15 	tar     && gem install --no-document fpm     && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs/ | bash -s -- -y     && PATH=$HOME/.cargo/bin:$PATH     && cargo install cargo-pgrx --version 0.8.3     && cargo pgrx init --${PG_VER}=$(which pg_config)     && git clone https://github.com/zombodb/zombodb.git     && cd ./zombodb     && cargo pgrx install --release     && cd ..     && rm -rf ./zombodb     && apk del .zombodb-build-deps' returned a non-zero code: 1
make: *** [Makefile:62: .build_main_pg15] Error 1

Error: Process completed with exit code 2.
```

[This CI](https://github.com/Samagra-Development/WarpSQL/actions/runs/5220763603/jobs/9424236595)

--- a/Dockerfile
+++ b/Dockerfile
@@ -249,7 +249,7 @@ RUN apk add --no-cache --virtual .zombodb-build-deps \
     && gem install --no-document fpm \
     && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | bash -s -- -y \
     && PATH=$HOME/.cargo/bin:$PATH \
-    && cargo install cargo-pgrx --version 0.8.3 \
+    && cargo install cargo-pgrx --version 0.9.3 \
     && cargo pgrx init --${PG_VER}=$(which pg_config) \
     && git clone https://github.com/zombodb/zombodb.git \
     && cd ./zombodb \

--- a/bitnami/Dockerfile
+++ b/bitnami/Dockerfile
@@ -234,7 +234,7 @@ RUN apt-get update -y -qq --fix-missing \
     && export PATH="/.cargo/bin:$PATH" \
     && export PGRX_HOME="/.pgrx/" \
     && mkdir -p $PGRX_HOME \
-    && cargo install cargo-pgrx --version 0.8.3 \
+    && cargo install cargo-pgrx --version 0.9.3 \
     && cargo pgrx init --pg${PG_MAJOR}=/opt/bitnami/postgresql/bin/pg_config \
     && git clone https://github.com/zombodb/zombodb.git \
     && cd zombodb \

